controllers/nutanixmachine_controller.go

```diff
@@ -856,3 +861,8 @@ func (r *NutanixMachineReconciler) addVMToProject(rctx *nctx.MachineContext, vmM
 	conditions.MarkTrue(rctx.NutanixMachine, infrav1.ProjectAssignedCondition)
 	return nil
 }
+
+func (r *NutanixMachineReconciler) isGetRemoteClientConnectionError(err error) bool {
+	const expectedErrString = "connect: connection refused"
```

Comment:
pls add comment stating possible reasons of this error

Reply:
Added comment